features/graph/api/src/main/java/org/opennms/netmgt/graph/api/generic/GenericProperties.java

```diff
@@ -51,4 +51,7 @@
 
     /** Reference to a node, either the id, or a <foreignSource>:<foreignId> statement */
     String NODE_REF = "nodeRef";
+
+    String NODE_INFO = "nodeInfo";
```

Comment:
hm, today it is set nowhere but exposed via NodeAware and therefor implemented by GenericVertex. How would it be computed?

Reply:
That part is not yet implemented.
This will be used as a contract by the Enrichment Engine, which later on will enrich/compute properties adhoc and not when building the graph. A NodeAware simply allows to determine for the Enrichment Engine that this vertex must be enriched with node data. 

If this is confusing, you can also just remove the interfaces `NodeAware` and `LocationAware`, as we will need to come back to this anyways.
Otherwise, simply return null here.

Reply:
I removed it